### DISCUSSION TITLE
[Feature] hierarchy navigation of library

### DISCUSSION THREAD

**Discussion by Torstein-Eide** [immich-app/immich#6142] [Feature Request]
### Status que

Currently all photos are listed as a flat "feed" of media files.

```
* 27. okt. 2023
* Image1
* Image2
* Image3
* 26. okt. 2023
* Image1
* Image2
* Image3
```

### folder hierarchy

I like to be able to navigate the library via clickable hierarchy, on the sidebar. 

```
Libary Upload/
└── 2023/
    └── 2023-01-01/
        └── 2023-01-01_00:01image.jpeg
Libary sorted (External)/
└── 2023/
    └── 2023-01-01/
        └── 2023-01-01_00:01image.jpeg
```

Currently I have:

* 19 Years folders
* 2248 Date folders

### Example for Synology Photostation

![image](https://github.com/immich-app/immich/assets/1884894/01a41875-903f-4947-a34f-c65a6b063d28)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2073/#5110/#5409
